crates/sui-config/Cargo.toml

```diff
@@ -24,13 +24,9 @@ prometheus.workspace = true
 narwhal-config.workspace = true
 sui-keys.workspace = true
 sui-protocol-config.workspace = true
-sui-storage.workspace = true
 sui-types.workspace = true
 workspace-hack.workspace = true
 
-[target.'cfg(msim)'.dependencies]
-sui-simulator.workspace = true
```

Comment:
@bmwill seems like all deps from sui-simulator gets pulled in by default still, is this safe to remove? or there is a way disable build this crate by default?

Reply:
If CI passes you should be good!